src/component/linechart/index.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import { AreaChart, Area, Tooltip, XAxis } from 'recharts';
-import { IMetric } from 'module/interface/metric';
+// import { IMetric } from 'module/interface/metric';
```

Comment:
FMI why are you doing this? Do you know you need the import in a bit again? Otherwise I would suggest to not create such dead code that you have to cleanup later again.

Reply:
This is a good call. I was wondering if I should delete the component and remove the library as well. It would save some time if we want to leverage this in the next two weeks. But it could easily not matter ever again 🤔

Reply:
If in doubt we have the code in the history. So when you can find the commit you can easily bring back the chunks of code you are interested in. Doesn't work well with a lot of different interconnected modules but reasonably well for snippets.